src/PrecedenceGraph.h

```diff
@@ -109,6 +109,43 @@ class RecursiveClauses : public AstAnalysis {
     bool computeIsRecursive(const AstClause& clause, const AstTranslationUnit& translationUnit) const;
 };
 
+/**
+ * Analysis pass mapping identifiers with relations and clauses.
+ */
+class RelationDetail : public AstAnalysis {
```

Comment:
The name doesn't seem to reflect its purpose. `RelationDetailCache`, perhaps, or `RelationInfoCache`?

Reply:
Good point, I'll change it to `RelationDetailCache`.